src/core/src/bootstrap/EnvLayer.py

```diff
@@ -428,6 +429,25 @@ def write_with_retry_using_temp_file(file_path, data, mode='w'):
                     else:
                         raise Exception("Unable to write to {0} (retries exhausted). Error: {1}.".format(str(file_path), repr(error)))
 
+        @staticmethod
+        def delete_artifacts_from_dir(dir_name, artifact_identifier, raise_if_delete_failed=False):
+            """ Clears all artifacts from given dir. NOTE: Uses artifact_identifier to determine the content to delete """
+            artifacts_to_remove = glob.glob(str(dir_name) + "/" + str(artifact_identifier))
+            for artifact in artifacts_to_remove:
+                try:
+                    os.remove(artifact)
+                except Exception as error:
+                    error_message = "Unable to delete files from directory [Dir={0}][File={1}][Error={2}][RaiseIfDeleteFailed={3}].".format(
```

Comment:
Check and resolve code coverage warnings We're risking regressing on coverage % wherever the warnings exist.

Reply:
Pending

Reply:
Added more tests to resolve code coevrage warnings, waiting for it to complete execution on the latest iteration

Reply:
Resolved